assignments/lambda-classes.js

```diff
@@ -1 +1,201 @@
+    }
+
+    graduate(){
+        if(this.grade > 70){
+            console.log('Congratulations you are eligible to graduate from Lambda')
+        }
+        else{
+            console.log('Sorry, you can\'t graduate yet,keep working on those assignmnets!')
+        }
+    }
+}
+
+class ProjectManager extends Instructor{
+    constructor(PM){
+        super(PM);
+        this.gradClassName = PM.gradClassName;
+        this.favInstructor = PM.favInstructor;
+    }
+    standUp(channel){
+        console.log(`${this.name} announces to channel, @${channel} standy times!`)
+    }
+    debugsCode(student, subject){
+        console.log(`${this.name} debugs ${student.name}'s code on ${subject}`)
+    }
+
+}
+
+const Estevan = new Person({
+    name : 'Estevan',
+    age: '22',
+    location: 'Riverside, CA'
+
+});
+
+const Mom = new Person({
+    name : 'Silvia',
+    age: '42',
+    location: 'San Bernardino, CA'
+
+});
+
+const Brother = new Person({
+    name : 'James',
+    age: '24',
+    location: 'Irvine, CA'
+
+});
+
+
+
+const Suthep = new Instructor({
+    name: 'Suthep',
+    age:'21',
+    location: 'Hemet, CA',
+    specialty: 'front-end',
+    favLanguage: 'HTML',
+    catchPhrase: 'Its Lit'
+});
+
+const Ryan = new Instructor({
+    name: 'Ryan',
+    age:'20-something',
+    location: 'CA',
+    specialty: 'full-stack',
+    favLanguage: 'JavaScript',
+    catchPhrase: 'Great Work Everyone'
+});
+
+const Josh = new Instructor({
+    name: 'Josh',
+    age:'30-something',
+    location: 'CA',
+    specialty: 'everything',
+    favLanguage: 'javascript',
+    catchPhrase: 'Big Boss'
+});
+
+const Tom = new Student({
+    name: 'Tom',
+    age: '19',
+    location: 'El Monte, CA',
+    previousBackground: 'Worked at Blaze Pizza',
+    className: 'FSW16',
+    favSubjects: ['Math','CS','English'],
+    grade: 65
+});
+
+const Mathew = new Student({
+    name: 'Mathew',
+    age: '19',
+    location: 'El Monte, CA',
+    previousBackground: 'Worked at Blaze Pizza',
+    className: 'FSW16',
+    favSubjects: ['Fishing','Math','CS'],
+    grade: 80
+});
+
+const Cecil = new Student({
+    name: 'Cecil',
+    age: '20s',
+    location: 'Texas',
+    previousBackground: 'coder',
+    className: 'FSW16',
+    favSubjects: ['nap-time','CS','web-dev'],
+    grade: 100
+});
+
+const Abdul = new ProjectManager({
+    name: 'Abdul',
+    age: '30-ish',
+    location:'Lambda CPU',
+    gradClassName: 'FSW',
+    favInstructor:'Josh'
+});
+
+const Emily = new ProjectManager({
+    name: 'Emily',
+    age: '??',
+    location:'Okhlahoma',
+    gradClassName: 'FSW',
+    favInstructor:'Tom Tarpey'
+});
+
+const Caitlin = new ProjectManager({
+    name: 'Caitlin',
+    age: '22',
+    location:'Lambda CPU',
+    gradClassName: 'FSW',
+    favInstructor:'Josh'
+});
+
+
+console.log(Cecil.grade);
+console.log(Emily.points(Mathew))
+console.log(Tom.grade)
+console.log(Tom.graduate())
```

Comment:
graduate would need to point to points bc grade is not passing a number. I saw that you tried to link the two, it's possible but would require more code.